package.json

```diff
@@ -176,6 +176,7 @@
     "serve-handler": "^6.1.5",
     "strip-ansi": "^6.0.1",
     "typescript": "^5.3.2",
+    "typescript-plugin-css-modules": "^5.1.0",
```

Comment:
Had to add the package to the root dependencies as I discovered TypeScript tries to find it there